### PR TITLE
New package: swayidle-1.1

### PR DESCRIPTION
**Pull request by Derriick**
Idle management daemon for Wayland

### CODE REVIEW

File: srcpkgs/swayidle/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'swayidle'
+pkgname=swayidle
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config wayland-devel wayland-protocols scdoc git"
```
Comment (Cogitri) [inline srcpkgs/swayidle/template]: I don't think wayland-devel actually is a hostmakedepend

**Comment by Derriick** [comment]
For aarch64, armv7l and armv6l I get the error `Dependency "wayland-client" not found`, but I already have wayland-devel and wayland-protocols in hostmakedepends. Does anyone have any ideas ?

I will also remove the "depends" line, it is useless.

**Comment by Cogitri** [comment]
Qh, you're mixing something up here:

*Hostmakedepends == dependencies installed on the build host, e.g. for binaries that are run during build like pkg-config

*Makedepends == *-devel for the target to link against

File: srcpkgs/swayidle/template (hunk 1 above)
Comment (Derriick) [reply srcpkgs/swayidle/template]: I think I need it for cross-compile. Without it, I had a problem with another package: https://github.com/void-linux/void-packages/pull/7222#issuecomment-454726995

Comment (Cogitri) [reply srcpkgs/swayidle/template]: Oh, right. I guess it's required for both hostmakedepends and makedepends

**Comment by Derriick** [comment]
Ok, so do I have to add "wayland-devel" to make-depends too ?
In fact, I thought it was supposed to be run during build.

**Comment by Cogitri** [comment]
Yes, the wayland-scanner binary is run during build, but wayland-client is linked against too

**Comment by Derriick** [comment]
It seems to be good now, thank you for your help !

**Pull request merged**